Packs/Base/Scripts/CreateIndicatorRelationship/CreateIndicatorRelationship.py

```diff
@@ -0,0 +1,196 @@
+                                                 entity_b_type=indicator.get('entity_b_type', ''),
+                                                 source_reliability=args.get('source_reliability', ''),
+                                                 reverse_name=args.get('reverse_relationship', ''),
+                                                 first_seen=args.get('first_seen', ''),
+                                                 description=args.get('description', '')
+                                                 ))
+    return relationships
+
+
+def create_relationships_with_args(args: dict) -> List[EntityRelation]:
+    relationships = []
+    entity_b_list = argToList(args.get('entity_b'))
+    for entity_b in entity_b_list:
+        relationships.append(create_relationship(name=args.get('relationship', ''), entity_a=args.get('entity_a', ''),
+                                                 entity_a_type=args.get('entity_a_type', ''),
+                                                 entity_b=entity_b,
+                                                 entity_b_type=args.get('entity_b_type', ''),
+                                                 source_reliability=args.get('source_reliability', ''),
+                                                 reverse_name=args.get('reverse_relationship', ''),
+                                                 first_seen=args.get('first_seen', ''),
+                                                 description=args.get('description', '')))
+    return relationships
+
+
+def create_relationship(name: str, entity_a: str, entity_a_type: str, entity_b: str, entity_b_type: str,
+                        source_reliability: str = '', reverse_name: str = '', first_seen: str = '',
+                        description: str = '') -> EntityRelation:
+    return EntityRelation(
+        name=name,
+        reverse_name=reverse_name,
+        entity_a=entity_a,
+        entity_a_type=entity_a_type,
+        entity_b=entity_b,
+        entity_b_type=entity_b_type,
+        source_reliability=source_reliability,
+        brand=BRAND,
+        fields={
+            "firstSeenBySource": first_seen or datetime.now().isoformat('T'),
+            "lastSeenBySource": datetime.now().isoformat('T'),
+            "description": description
+        }
+    )
+
+
+# --------------------------------------------------- Main functions---------------------------------------------
+
+
+def validate_arguments(args: dict) -> Dict[str, str]:
+    """Get the args of the command and validate the arguments.
+
+    :return: raise an error if one of the validations fail.
+    :rtype: ``None``
+    """
+    if len(argToList(args.get('entity_a'))) > 1:
+        raise Exception("entity_a is a list, Please insert a single entity_a to create the relationship")
+    if len(argToList(args.get('entity_b_type'))) > 1:
+        raise Exception("entity_b_type is a list, Please insert a single type to create the relationship")
```

Comment:
should we also add check for
`len(argToList(args.get('entity_a_type'))) > 1`?

Reply:
sure, why not, added